### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by ghreza-crypto**
The project was created with `React Js` and linters added to it.

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi @ghreza-crypto , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+- **[using its calculator]**
+- **[single page application which makes using easy]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[mobile version functionality]**
```
Comment (1GORDON) [inline README.md:138]: Great job @ghreza-crypto 🤝

- [ ] [Optional] Kindly ensure this feature is relevant to this project. I would suggest you add a feature like `Add a calculator that uses Single Page Application`.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+- GitHub: [@githubhandle](https://github.com/ghreza-crypto)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing
+
+ Contributions, issues, and feature requests are welcome!
+- Feel free to check the [issues page](https://github.com/ghreza-crypto/to-do-list/issues)
```
Comment (1GORDON) [inline README.md:159]: Great job @ghreza-crypto 🤝

- [ ] [Optional] The issue link seems to be directed to a different project. Kindly ensure the issues listed here are specifically for this project.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank from Microverse and all of my teammates.
+
+- [Microverse Videos](https://www.youtube.com/watch?v=PrIY8sYwe90&ab_channel=Microversevideos)
+- [Setup Linters](https://questions.microverse.org/t/configure-linters-for-html-and-css/2009)
+- [Mobile First](https://www.mightyminnow.com/2013/11/what-is-mobile-first-css-and-why-does-it-rock/)
+- [Figma Templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
```
Comment (1GORDON) [inline README.md:179]: Great job @ghreza-crypto 🤝

- [ ] [Optional] Kindly ensure these acknowledgments are relevant to this project. You may remove the `Mobile First`, and `Figma` links from this section.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @ghreza-crypto ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**